packages/scoped-elements/src/shadyTemplateFactory.js

```diff
@@ -0,0 +1,47 @@
+import { templateCaches } from 'lit-html/lib/template-factory.js';
+import { marker, Template } from 'lit-html/lib/template.js';
+
+const getTemplateCacheKey = (type, scopeName) => `${type}--${scopeName}`;
+
+let compatibleShadyCSSVersion = true;
+
+// @ts-ignore
+const { ShadyCSS } = window;
+
+if (typeof ShadyCSS === 'undefined') {
+  compatibleShadyCSSVersion = false;
+} else if (typeof ShadyCSS.prepareTemplateDom === 'undefined') {
+  compatibleShadyCSSVersion = false;
+}
+
+/**
+ * Template factory which scopes template DOM using ShadyCSS.
+ * @param scopeName {string}
+ */
+export const shadyTemplateFactory = scopeName => result => {
```

Comment:
It would be great if lit-html exported this function. By the moment this is a workaround to continue supporting shadyCSS

Reply:
Can we file a feature request at lit-html to expose this?

Reply:
https://github.com/Polymer/lit-html/issues/1134

Reply:
Great!